Dockerfile.build

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:14.04
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true \
+    | /usr/bin/debconf-set-selections
+
+RUN apt-get update -q \
+    && apt-get install -y --no-install-recommends \
+        software-properties-common \
+        python-software-properties \
+    && apt-get update -q \
+    && add-apt-repository ppa:webupd8team/java
+
+RUN apt-get update -q \
+    && apt-get install -y --no-install-recommends \
+        build-essential \
+        ca-certificates \
+        curl \
+        git \
+        libbz2-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        libssl-dev \
+        zlib1g-dev \
+        jq \
+        moreutils \
+        gettext \
+        unzip \
+        oracle-java8-installer \
+        oracle-java8-set-default
+
+RUN useradd -d /home/circle -ms /bin/bash -g root -G sudo circle
+USER circle
+WORKDIR /home/circle
+
+ENV PYENV_ROOT /home/circle/pyenv
+ENV PATH ${PYENV_ROOT}/shims:${PYENV_ROOT}/bin:${PATH}
+ENV PYTHONDONTWRITEBYTECODE true
+ENV PYENV_VERSION 3.6.3
+RUN mkdir -p ${PYENV_ROOT} \
+    && cd ${PYENV_ROOT} \
+    && git clone https://github.com/yyuu/pyenv.git . \
+    && git checkout --quiet v1.2.4 \
+    && echo 'eval "$(pyenv init -)"' >> ${HOME}/.bashrc
+RUN pyenv install ${PYENV_VERSION}
+RUN pip install --upgrade pip==10.0.1
+RUN pip install virtualenv==16.0.0
+
+ENV PATH /home/circle/make4/usr/bin:${PATH}
+RUN curl --silent --remote-name \
+        http://us-east-1.ec2.archive.ubuntu.com/ubuntu/pool/main/m/make-dfsg/make_4.1-6_amd64.deb \
+    && mkdir -p /home/circle/make4 \
+    && dpkg -x make*.deb /home/circle/make4
```

Comment:
As we discussed, it would be nice (but not urgent) to get rid of this and go to 18.04. Could you file an issue/task to track that?

Reply:
https://github.com/HumanCellAtlas/data-store/issues/1293